lib/model/localizations.dart

```diff
@@ -15,3 +17,60 @@ abstract final class GlobalLocalizations {
   static ZulipLocalizations zulipLocalizations =
     lookupZulipLocalizations(ZulipLocalizations.supportedLocales.first);
 }
+
+/// The mapping, of the locales we offer, to their display name in their own
+/// locale a.k.a. selfname.
+///
+/// [ZulipLocalizations.supportedLocales] can include languages that only have
+/// a few strings translated.  This map should only inlcude sufficiently
+/// translated locales from [ZulipLocalizations.supportedLocales].
```

Comment:
```suggestion
/// This map only includes some of [ZulipLocalizations.supportedLocales];
/// it includes languages that have substantially complete translations.
/// For what counts as substantially translated, see docs/translation.md.
```

(and then make sure docs/translation.md answers the question)